portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  overflow-x: hidden;
```

Comment:
Reorder to be alphabetical order. ditto to the lines below.

Reply:
done.